app/models/course/assessment/submission.rb

```diff
@@ -193,12 +194,18 @@ def validate_unique_submission
     end
   end
 
-  def send_notification
+  def send_attempt_notification
     return unless course_user.real_student?
 
     Course::AssessmentNotifier.assessment_attempted(creator, assessment)
   end
 
+  def send_submit_notification
+    return unless course_user.real_student? && workflow_state_changed?
```

Comment:
To be safe, better check it's from attempting to submitted.  use `workflow_state_was`